.gitignore

```diff
@@ -72,6 +72,9 @@ debug.log
 # Cscope
 cscope.*
 
+# Ctags
+tags
```

Comment:
not related to the PR ?

Reply:
No indeed, does it need to have its own PR ?

Reply:
Yes please.

Reply:
Feel free to reference this PR to indicate that it's related.